lib/app-common.ts

```diff
@@ -115,21 +117,36 @@ export interface GlobalTunnelNgConfig {
  * default exclusion patterns are added for all private IPv4 address ranges.
  */
 async function setupGlobalHttpProxy(settings: CliSettings) {
-	const semver = await import('semver');
-	if (semver.lt(process.version, '10.16.0')) {
-		setupGlobalTunnelNgProxy(settings);
-	} else {
-		// use global-agent instead of global-tunnel-ng
-		await setupGlobalAgentProxy(settings);
+	// `global-tunnel-ng` accepts lowercase variables with higher precedence
+	// than uppercase variables, but `global-agent` does not accept lowercase.
+	// Set uppercase versions for backwards compatibility.
+	const { env } = process
```

Comment:
Build failing because prettier was not run (missing semicolon and other errors).

Reply:
Yeah.. I'm used to every other repo running it automatically :P

Reply:
Fixed for each commit